osafw-app/App_Code/fw/FwConfig.cs

```diff
@@ -151,11 +151,11 @@ private static void specialSettings()
         if (!settings.ContainsKey("ASSETS_URL"))
             settings["ASSETS_URL"] = settings["ROOT_URL"] + "/assets";
 
-        // default template dir
-        if (!settings.ContainsKey("template"))
```

Comment:
I don't see a need to rename template to template_theme

Reply:
> I don't see a need to rename template to template_theme

I think I changed the setting name for more clearness and theming emphasizing. In case of several themes in the project it's clear from the config that we're talking about themes and what specific themes we have:

    "template_theme": "hyper",
    "template_theme1": "default",
    "template_theme2": "falcon",
    "template_theme3": "hyper",
    "is_select_template_theme": true,

Of course, it's not the case for the default framework codebase without the "template" or "template_theme" config option at all, or a particular project with just one theme. It should be no harm.